Human Travis says:

> [ 56%] Building CXX object tests/Unit/Elliptic/DiscontinuousGalerkin/CMakeFiles/Test_EllipticDG.dir/Test_ImposeBoundaryConditions.cpp.o
g++: interner Compiler-Fehler: Speicherzugriffsfehler (Programm cc1plus)
Bitte senden Sie einen vollständigen Fehlerbericht auf Englisch ein;
inclusive vorverarbeitetem Quellcode, wenn es dienlich ist.
Weitere Hinweise finden Sie unter »<https://gcc.gnu.org/bugs/>«.
make[3]: *** [tests/Unit/Elliptic/DiscontinuousGalerkin/CMakeFiles/Test_EllipticDG.dir/Test_ImposeBoundaryConditions.cpp.o] Fehler 4
make[2]: *** [tests/Unit/Elliptic/DiscontinuousGalerkin/CMakeFiles/Test_EllipticDG.dir/all] Fehler 2
make[1]: *** [CMakeFiles/test-executables.dir/rule] Fehler 2
make: *** [test-executables] Fehler 2

I am not able to compile on Minerva due to some internal compiler error. °~°

--- a/cmake/AddCxxFlag.cmake
+++ b/cmake/AddCxxFlag.cmake
@@ -10,8 +10,8 @@ function(check_and_add_cxx_flag FLAG_TO_CHECK)
   execute_process(
     COMMAND
     bash -c
-    "${CMAKE_CXX_COMPILER} -Werror ${POSITIVE_FLAG_TO_CHECK} -x c++ -c \
-- <<< \"\" -o /dev/null"
+    "LC_ALL=POSIX ${CMAKE_CXX_COMPILER} -Werror ${POSITIVE_FLAG_TO_CHECK} \
+-x c++ -c - <<< \"\" -o /dev/null"
     RESULT_VARIABLE RESULT
     ERROR_VARIABLE ERROR_FROM_COMPILATION
     OUTPUT_QUIET)
@@ -38,8 +38,8 @@ function(check_and_add_cxx_flags FLAGS_TO_CHECK)
   execute_process(
     COMMAND
     bash -c
-    "${CMAKE_CXX_COMPILER} -Werror ${POSITIVE_FLAGS_WITH_SPACES} -x c++ -c \
-- <<< \"\" -o /dev/null"
+    "LC_ALL=POSIX ${CMAKE_CXX_COMPILER} -Werror ${POSITIVE_FLAGS_WITH_SPACES} \
+-x c++ -c - <<< \"\" -o /dev/null"
     RESULT_VARIABLE RESULT
     ERROR_VARIABLE ERROR_FROM_COMPILATION
     OUTPUT_QUIET)
@@ -49,14 +49,8 @@ function(check_and_add_cxx_flags FLAGS_TO_CHECK)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${FLAGS_WITH_SPACES}" PARENT_SCOPE)
   else(${RESULT} EQUAL 0)
     # Check each flag to see if it was marked as "invalid" in the output
-    string(FIND "${ERROR_FROM_COMPILATION}" "‘" GCC_STYLE_QUOTE)
     foreach(FLAG ${POSITIVE_FLAGS_TO_CHECK})
-      if(NOT ${GCC_STYLE_QUOTE} EQUAL -1)
-        string(FIND "${ERROR_FROM_COMPILATION}" "‘${FLAG}’" FOUND_POS)
-      else()
-        string(FIND "${ERROR_FROM_COMPILATION}" "'${FLAG}'" FOUND_POS)
-      endif()
-
+      string(FIND "${ERROR_FROM_COMPILATION}" "'${FLAG}'" FOUND_POS)
       if(${FOUND_POS} EQUAL -1)
         # For some reason:
         # list(FIND ${POSITIVE_FLAGS_TO_CHECK} ${FLAG} INDEX_OF_FLAG)

--- a/support/Environments/minerva_gcc.sh
+++ b/support/Environments/minerva_gcc.sh
@@ -4,7 +4,7 @@
 # See LICENSE.txt for details.
 
 spectre_setup_modules() {
-    echo "All modules on Minerva are provided by the system"
+    source /home/SPACK/share/spack/setup-env.sh
 }
 
 spectre_unload_modules() {
@@ -19,7 +19,11 @@ spectre_unload_modules() {
     module unload hdf5-1.10.1-gcc-7.3.0-w32z4ik
     module unload jemalloc-4.5.0-gcc-7.3.0-wlf2m7r
     module unload libxsmm-1.9-gcc-7.3.0-npctj4l
+    module unload libsharp-2018-01-17-gcc-7.3.0-4xamgaw
     module unload openblas-0.3.0-gcc-7.3.0-cjwkbox
+    module unload python-2.7.15-gcc-7.3.0-2azrjn6
+    module unload py-numpy-1.13.3-gcc-7.3.0-ggad2xs
+    module unload py-scipy-1.0.0-gcc-7.3.0-tegl6xx
     module unload yaml-cpp-develop-gcc-7.3.0-qcfbbll
 }
 
@@ -34,8 +38,12 @@ spectre_load_modules() {
     module load gsl-2.4-gcc-7.3.0-yobvdek
     module load hdf5-1.10.1-gcc-7.3.0-w32z4ik
     module load jemalloc-4.5.0-gcc-7.3.0-wlf2m7r
+    module load libsharp-2018-01-17-gcc-7.3.0-4xamgaw
     module load libxsmm-1.9-gcc-7.3.0-npctj4l
     module load openblas-0.3.0-gcc-7.3.0-cjwkbox
+    module load python-2.7.15-gcc-7.3.0-2azrjn6
+    module load py-numpy-1.13.3-gcc-7.3.0-ggad2xs
+    module load py-scipy-1.0.0-gcc-7.3.0-tegl6xx
     module load yaml-cpp-develop-gcc-7.3.0-qcfbbll
 }
 

